packages/frontend/src/lib/disk-image/DiskImageDetailsVirtualMachine.svelte

```diff
@@ -187,7 +187,14 @@ async function launchVM(build: BootcBuildInfo): Promise<void> {
   // To avoid a blank terminal wait until terminal has logs and and then show it
   // logs.terminal.buffer.normal will contain the "ascii cursor" with a value of 1 until there is more logs.
   // we wait until buffer.normal.length is more than 1.
+  const startTime = Date.now();
+  const timeout = 30000; // 30 seconds
```

Comment:
```suggestion
  const timeout = 30_000; // 30 seconds
```

suggestion: for readability, using _ is easier to read